examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter

```diff
@@ -294,17 +300,35 @@ server cluster OnOff = 6 {
     kAcceptOnlyWhenOn = 0x1;
   }
 
-  readonly attribute boolean onOff = 0;
+  readonly nosubscribe attribute boolean onOff = 0;
```

Comment:
Again, this should not be "nosubscribe".

Especially since the XML says `reportable="true"`!  ZAP is just really broken here, afaict.

Reply:
changed.